theatre/core/src/propTypes/index.ts

```diff
@@ -690,6 +729,9 @@ export interface PropTypeConfig_StringLiteral<T extends string>
 
 export interface PropTypeConfig_Rgba extends ISimplePropType<'rgba', Rgba> {}
 
+export interface PropTypeConfig_Image
+  extends ISimplePropType<'image', string> {}
```

Comment:
It'll be hard to distinguish between image and a simple string. Use `{type: 'Image', path: string}` instead

Reply:
Using {type: 'Image', path: ''} instead of a simple string type would make it the first prop type to somehow encode the theatre type in the value instead of the theatre type being determined at runtime through sheet.object() calls. I'd be curious to hear more about this idea before committing to it.

Reply:
Answered [here](https://github.com/theatre-js/theatre/pull/363#discussion_r1056972282).